OracleIdentityGovernance/samples/containerizedDB/docker-compose.yaml

```diff
@@ -68,14 +77,17 @@ services:
     environment:
       - ADMIN_HOST=${DC_HOSTNAME}
       - ADMIN_PORT=7001
-      - adminhostname=${DC_HOSTNAME}
-      - adminport=7001
+      - ADMIN_PASSWORD=${DC_ADMIN_PWD}
       - MANAGED_SERVER=soa_server1
-      - server=soa_server1
+      - MS_HOST=${DC_HOSTNAME}
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://soams:8001/soa-infra"]
```

Comment:
If you're allowing the hostname to change, wouldn't this test need to change as well?